CODEOWNERS

```diff
@@ -58,6 +59,7 @@
 /soc/arm64/qemu_cortex_a53/               @carlocaione
 /soc/arm64/bcm_vk/                        @abhishek-brcm
 /soc/arm64/xenvm/                         @lorc
+/soc/arm64/arm/                           @povergoing
```

Comment:
Guess you would like to move all ARM64 SoCs here, so maybe add @carlocaione as well?

Reply:
I don't think that's a good idea. The ARM64 SoCs are in the `soc/arm64/` directory, while `soc/arm64/arm` is for FVP only.

Reply:
alright, i probably missed that.